src/pytest_container/container.py

```diff
@@ -280,6 +287,17 @@ def cleanup(self) -> None:
             )
             assert self.host_path
             self._tmpdir.cleanup()
+        else:
+            # Clean up container volume
+            if not os.path.isabs(self.host_path):
```

Comment:
mypy complaint: Argument 1 to "isabs" has incompatible type "Optional[str]"; expected "Union[str, bytes, PathLike[str], PathLike[bytes]]"